TourMap/Utilities/Constants.swift

```diff
@@ -16,11 +16,26 @@ struct Constant {
     static let onboardingDescription1 = "\"Few places in this world are more dangerous than home. Fear not, therefore, to try the mountain passes. They will kill care, save you from deadly apathy, set you free, and call forth every faculty into vigorous, enthusiastic action.\" ~John Muir"
     static let onboardingDescription2 = "\"It seems that the more places I see and experience, the bigger I realize the world to be. The more I become aware of, the more I realize how relatively little I know of it, how many places I have still to go, how much more there is to learn.\" ~Anthony Bourdain"
     static let onboardingDescription3 = "\"A journey of a thousand miles begins with a single step.\" ~Lao Tzu There’s a sunrise and sunset every single day, and they’re absolutely free. Don’t miss so many of them.“Man cannot discover new oceans unless he has the courage to lose sight of the shore.” ~Andre Gide"
+    static let buttonHeight = CGFloat(50)
+  }
+  
+  struct K {
```

Comment:
can you rename this to something more readable... what does struct k mean?

Reply:
I made the necessary changes.